oeps/architectural-decisions/oep-0050-hooks-extension-framework.rst

```diff
@@ -276,3 +281,6 @@ Adding an official arbiter
 
 17 April 2021 - Felipe Montoya
 Updating based on feedback from the community and edX arch team.
+
+20 July 2022 - Maria Grimaldi
```

Comment:
Nit: Change history should should be ordered most recent first. See https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#change-history

Reply:
I didn't know that! Thanks!